assets/source/setup-guide/app/components/Billing/Billing.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { __ } from '@wordpress/i18n';
+import { Spinner } from '@woocommerce/components';
+import { recordEvent } from '@woocommerce/tracks';
+import {
+	Button,
+	CardBody,
+	Flex,
+	FlexItem,
+	FlexBlock,
+	__experimentalText as Text, // eslint-disable-line @wordpress/no-unsafe-wp-apis --- _experimentalText unlikely to change/disappear and also used by WC Core
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/**
+ * Clicking on "Setup Billing" button.
+ *
+ * @event wcadmin_pfw_billing_setup_button_click
+ */
+
+/**
+ * Clicking on "Go to billing settings" button.
+ *
+ * @event wcadmin_pfw_go_to_billing_button_click
+ */
+
+const Billing = ( { accountData } ) => {
+	const isBillingSetup = accountData?.is_billing_setup;
+	const statusLabe = classnames( 'pfw-billing-info', {
+		'pfw-billing-info--status-success': isBillingSetup === true,
+		'pfw-billing-info--status-error': isBillingSetup === false,
+	} );
+
+	let element = null;
```

Comment:
I'd prefer not to use `let element` and assign JSX to it. 

I'd prefer something like this:

```js
	const getElement = () => {
		if (isBillingSetup === undefined) {
			return "spinner here";
		}

		if (isBillingSetup === false) {
			return "no valid billing here";
		}

		// when isBillingSetup === true
		return "true condition";
	};

	// then when render:
	<Flex direction="row" className="pfw-billing-info">
		{getElement()}
	</Flex>;
```

With this, we don't need to keep track of what is `element`.